src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -994,4 +982,18 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
             return forbidden();
         }
     }
+
+    /**
+     * GET /courses/{courseId}/achievements : Get all achievements for course.
+     *
+     * @param courseId the course to which the achievements belong to
+     * @return the ResponseEntity with status 200 (OK) and with the found achievements as body
+     */
+    @GetMapping("/courses/{courseId}/achievements")
```

Comment:
I would move that to `AchievementResource.java`

Reply:
This was also my first thought, but looking at the URL wouldn´t it make more sense in the `CourseResource.java` since I am loading sth. for the course?